core/src/test/java/org/elasticsearch/index/query/BaseQueryTestCase.java

```diff
@@ -116,7 +122,10 @@ public static void init() throws IOException {
                         Version.V_1_0_0, Version.CURRENT))
                 .build();
 
-        index = new Index("test");
+        index = new Index(randomAsciiOfLengthBetween(1, 10));
+        final TestClusterService clusterService = new TestClusterService();
+        clusterService.setState(new ClusterState.Builder(clusterService.state()).metaData(new MetaData.Builder().put(
+                new IndexMetaData.Builder(index.name()).settings(Settings.builder().put(IndexMetaData.SETTING_VERSION_CREATED, Version.CURRENT)).numberOfShards(1).numberOfReplicas(0))));
```

Comment:
sure we can do that, I didn't remember we were picking a version randomly to be honest, will look into that ;)